lxd/storage/storage.go

```diff
@@ -170,11 +170,11 @@ func UsedBy(ctx context.Context, s *state.State, pool Pool, firstOnly bool, memb
 					}
 				} else {
 					// Handle orphaned image volumes that are not associated to an image.
-					u = vol.URL(version.APIVersion, pool.Name())
+					u = vol.URL(version.APIVersion)
```

Comment:
These sorts of changes as well as the function arg change itself should be a separate commit, preceded by a commit for the db change that allows, and then finally a commit to add the new api handler.

So a commit per logical change:

1. API extension
1. API struct change
2. DB function change to populate new field
3. Change to URL function to remove unnecessary argument
4. New api handler endpoint
5. Client package changes
6. CLI command changes